src/App.tsx

```diff
@@ -1,14 +1,35 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
-import { TodoList } from './components/TodoList';
+import { TodoList, StatusOfFilter } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { isInQuery } from './helpers/is-in-query';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodoId, setSelectedTodoId] = useState(0);
+  const [userId, setUserId] = useState(0);
+  const [status, setStatus] = useState<StatusOfFilter>(StatusOfFilter.All);
+  const [query, setQuery] = useState('');
+
+  const todo = useMemo(() => todos.find(item => (item.id === selectedTodoId)) as Todo, [selectedTodoId]);
+
+  const visibleTodos = useMemo(() => todos.filter(item => isInQuery(item.title, query)), [todos, query]);
+
+  const handleChange = (value: StatusOfFilter) => {
```

Comment:
Consider renaming the function 'handleChange' to 'handleStatusChange' for better clarity.